silkworm/db/datastore/snapshots/history_range_by_keys_query.hpp

```diff
@@ -70,7 +70,7 @@ struct HistoryRangeByKeysSegmentQuery {
         return std::pair{std::move(key_data), std::move(*value_opt)};
     }
 
-    auto exec_with_eager_begin(Bytes key_start, Bytes key_end, datastore::Timestamp timestamp, bool ascending) {
+    auto exec_with_eager_begin(const Bytes& key_start, Bytes key_end, datastore::Timestamp timestamp, bool ascending) {
```

Comment:
I recommend to add NOLINT here or pack keys into std::pair in order to preserve symmetry.